src/services/user/services/AdminUsers.js

```diff
@@ -248,6 +248,35 @@ class AdminUsers {
 		}
 	}
 
+	async markUserAsDeleted(userId) {
+		if (userId) {
+			await this.app.service('users').remove(null, {
+				query: {
+					userId,
+					deletedAt: {
+						default: new Date(),
+						// after one week
+						expireAfterSeconds: 30,
```

Comment:
30 seconds are not a week?

Reply:
and I think you missmatch this with the ttl index: https://docs.mongodb.com/manual/tutorial/expire-data/